### PR TITLE
Fixes #142, implements convert function for DepthWiseConv2D and relu6 activation

### LINKED ISSUE #142: Conversion of Keras MobileNet fails
The Following code fails.

    from keras.applications.mobilenet import MobileNet
    model = MobileNet(include_top=True, weights='imagenet')

    from onnxmltools import convert_keras
    onx = convert_keras(model, 'mobilenet.onnx')
    
    from onnxmltools.utils import save_model
    save_model(onx, "mobilenet.onnx")

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: onnxmltools/convert/common/_apply_operation.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def _apply_unary_operation(scope, op_type, input_name, output_name, container, o
     name = _create_name_or_use_existing_one(scope, op_type, operator_name)
 
     attrs['name'] = name
-    if container.targeted_onnx_version <= StrictVersion('1.0'):
+    if compare_strict_version(container.targeted_onnx_version, '1.0') <0:
```
Comment (wenbingl) [inline onnxmltools/convert/common/_apply_operation.py]: space

File: onnxmltools/convert/common/_topology.py [hunk 2]
```diff
@@ -256,7 +256,10 @@ def __init__(self, model, default_batch_size=1, initial_types=None,
         self.initial_types = initial_types if initial_types else list()
         self.metadata_props = metadata_props if metadata_props else dict()
         self.default_batch_size = default_batch_size
-        self.targeted_onnx_version = StrictVersion(targeted_onnx)
+        if targeted_onnx is None:
```
Comment (wenbingl) [inline onnxmltools/convert/common/_topology.py]: the condition statement looks redundant.

File: onnxmltools/convert/common/_topology.py [hunk 3]
```diff
@@ -634,7 +637,7 @@ def convert_topology(topology, model_name, doc_string, targeted_onnx):
     include '1.1.2', '1.2', and so on.
     :return: a ONNX ModelProto
     '''
-    if targeted_onnx != onnx.__version__:
+    if targeted_onnx is not None and compare_strict_version(targeted_onnx, onnx.__version__) != 0:
```
Comment (wenbingl) [inline onnxmltools/convert/common/_topology.py]: 'is not None' is redundant?

**Comment by wenbingl** [comment]
It looks this PR contains other commits from the merged one?

**Comment by xadupre** [comment]
>           It looks this PR contains other commits from the merged one?

Yes, but the diff should be readable now.

**Pull request merged**